tmt/utils.py

```diff
@@ -98,7 +98,7 @@ def relative_to(self, other: Union[str, 'Path']) -> 'Path':  # type: ignore[over
     # implementation considers to not be relative to each other. Therefore, we
     # need to override `is_relative_to()` even for other Python versions, to not
     # depend on `ValueError` raised by the original `relative_to()`.
-    def is_relative_to(self, other: 'Path') -> bool:
+    def is_relative_to(self, other: 'Path') -> bool:  # type: ignore[override]
         # NOTE: the following is not perfect, but it should be enough for
```

Comment:
@happz With py39+ in mind, couldn't the way `relative_to` and `is_relative_to` is being used be changed? (would help with the mypy error)

Reply:
I don't think so. The method was a missing piece with Python 3.8 and older, but it's still important with 3.9+ as the one present in `pathlib.Path` behaves in a way tmt code does not expect. The version here is a red herring, it's not an indication this method should be removed along with Python 3.8 and older.

Reply:
Right, I meant more like what it would take to make tmt expect the pathlib behaviour. Apologies, I don't know the code that well yet.

Reply:
Aha, got it. No idea, TBH, as relative paths and their detection in the library-handling code were making me sad, and things were too complicated for me to realize a clear path forward. But making tmt more compatible with standard library is definitely a noble goal.

Reply:
I'm going to mark this conversation as resolved, we can definitely address this issue later - I filed https://github.com/teemtee/tmt/issues/2253 to track this concern.